models/containerized_model/README.md

```diff
@@ -0,0 +1,29 @@
+# How to create containerized models?
+
+## Pre-requisites
+
+- Certifai toolkit (from [CognitiveScale website](https://www.cognitivescale.com/try-certifai/)).
+- A model (pickle) stored in a cloud storage (e.g Amazon S3).
+- A base docker image which (strictly) simulates the trained model python environment.
+
+
+## Step 1 - Template generation
+TODO:
+- This should ask for full docker base image path (and inject into the Dockerfile) and generate the code template
+
+
+## Step 2 - Copy artifacts
+Copy the `packages` folder from inside the toolkit to current directory (`containerized_model` in this case)
+
+## Step 3 - Configure cloud storage
+- Add respective cloud storage credentials and `MODEL_PATH` to `environment.yaml` file.
+
+## Step 4 - Build and run
```

Comment:
Separate build and run. Run is where you'd have to inject the right environment variables.

Reply:
Environment variables are being injected at runtime, but I just thought this is a cleaner way to put them in a file and then do a run instead of crunching so many env vars in the run command. Just syntactic sugar and looks neat. I’ll update the read me with more details and let me know if you think that would make more sense.